### ISSUE TITLE
Oracle driver SHA256 mismatch

### ISSUE THREAD

**Issue by cm325** [OSGeo/homebrew-osgeo4mac#191]
Hi, I'm getting this error trying to install:

Installing qgis2 from osgeo/osgeo4mac
==> Installing dependencies for osgeo/osgeo4mac/qgis2: oracle-client-sdk, pyspatialite
==> Installing osgeo/osgeo4mac/qgis2 dependency: oracle-client-sdk
==> Downloading file:///Users/myusername/Library/Caches/Homebrew/instantclient-basiclite-macos.x64-11.2.0.4.0.zip
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 8ebc4baafb40a3b7145d48a4be86747766fdb3e1e0ac7cdd397e852164fa2189
Actual: d51c5fb67d1213c9b3c6301c6f73fe1bef45f78197e1bae7804df4c0abb468a7
Archive: /Users/myusername/Library/Caches/Homebrew/oracle-client-sdk-11.2.0.4.0.zip
To retry an incomplete download, remove the file above.

have the files changed?

**Comment by dakcarto** [comment]
Hi @cm325, thanks for reporting.

Formula updated to version 12.1.0.2.0 in 6e760b7. Please reopen issue if necessary.

**Comment by chrismarx** [comment]
Thanks it works!!

**Comment by chrismarx** [comment]
Hmm, well that solved the build problem. But now when I try to make a connection, qgis just crashes. I added error logging as mentioned here (http://gis.stackexchange.com/questions/84790/where-does-qgis-write-its-error-log-messages) and all I got in the error log is:

QOCISpatialDriver: unable to create environment

I assumed that maybe I was entering the connection information improperly, but I just installed the latest version of osgeo4w's version of qgis on a window virtual machine, running on the same computer, and entered the exact same connection information, and worked perfectly.
